.gitignore

```diff
@@ -1 +1,3 @@
 build/**
+.build
+Package.resolved
```

Comment:
We might not want to ignore `Package.resolved`

Reply:
Should it be checked in?

Reply:
yea